src/elements/content-picker/ContentPicker.js

```diff
@@ -759,8 +761,8 @@ class ContentPicker extends Component<Props, State> {
      * @param {Object} item file or folder object
      * @return {void}
      */
-    select = (item: BoxItem): void => {
-        const { type: selectableType, maxSelectable }: Props = this.props;
+    select = (item: BoxItem, forceSharedLink: boolean = true): void => {
```

Comment:
Can we make the latter argument an options object so we can use named parameters? Otherwise it's difficult to tell what passing `false` does here by looking at only the calling code.